include/components/config.hpp

```diff
@@ -356,6 +367,11 @@ class config {
   string m_file;
   string m_barname;
   sectionmap_t m_sections{};
+
+  /**
+   * All files parsed while parsing this config (actual config file excluded)
```

Comment:
I agree that single-line comments inside functions should use `//`, I changed my code to that. But I think member declarations should use `/** */` comments, because we would need to use `///` comments for it to be recognized by doxygen, which would add another type of comment to the code base.